addons/wasmedge/enable

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SNAP_CHANNEL=--candidate
+PATCH_DIR=`dirname "$0"`/patch
+
+echo "Enabling wasmedge via crun OCI runtime"
+
+echo "Install/Upgrade crun-wasmedge from snapstore"
+[ ! -d "/snap/crun-wasmedge/current" ] && sudo snap install $SNAP_CHANNEL crun-wasmedge || sudo snap refresh $SNAP_CHANNEL crun-wasmedge
+sudo ln -sf /snap/crun-wasmedge/current/crun.wrapper /usr/local/bin/crun
```

Comment:
Do you really need this link? Snaps intend to be agnostic of the host they run on. When we move to a strictly confined snap this will become an issue.

Reply:
No, the link is not needed. It was just a quick hack.